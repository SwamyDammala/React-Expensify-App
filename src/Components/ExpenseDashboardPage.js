import React from 'react'
import ExpenseList from './ExpenseList'
import ExpenseListFilter from './ExpenseListFilter'
import ExpenseSummary from './ExpenseSummary'
import DeleteModal from './deleteModal'

const ExpenseDashboardPage=()=>(
    <div>
        <ExpenseSummary />
        <ExpenseListFilter />
        <ExpenseList />
    </div>    
)

export default ExpenseDashboardPage