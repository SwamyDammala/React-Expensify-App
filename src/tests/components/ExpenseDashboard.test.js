import React from 'react'
import {shallow} from 'enzyme'
import ExpenseDashboardPage from '../../Components/ExpenseDashboardPage'


test('Should Test Expennse DashboardPage',()=>{
    const wrapper=shallow(<ExpenseDashboardPage />)
    expect(wrapper).toMatchSnapshot()
})