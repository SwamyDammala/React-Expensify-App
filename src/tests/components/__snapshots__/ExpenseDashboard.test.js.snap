// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Test Expennse DashboardPage 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
</div>
`;
